src/arrays.js

```diff
@@ -1,59 +1,77 @@
 export const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length) {
+    return array[index % array.length]
+  } 
+  return array[index]
 };
 
 export const arrayToCSVString = (array) => {
-  // your code here
+  return array.toString()
 };
 
 export const csvStringToArray = (string) => {
-  // your code here
+  return string.split(',')
 };
 
-export const addToArray = (array, element) => {
-  // your code here
+export const addToArray = (element, array) => {
+  array.push(element)
 };
 
-export const addToArray2 = (array, element) => {
-  // your code here
+export const addToArray2 = (element, array) => {
+  const array2 = array.concat(element) 
+  return array2
```

Comment:
Unless a `return` statement is the only line inside a block, it's usually nice to have a line break before it so it stands out :)